### ISSUE TITLE
BEDGRAPH input requirements

### ISSUE THREAD

**Issue by CriticalSci** [SvenBaileyLab/HisTrader#3]
Hello!

Could you clarify what kind of BEDGRAPH files this tool expects?

I tried generating some using bedtools genomecov but have been getting errors where the index is out of range.

Now I'm considering MACS2 but this also outputs several types of bedgraphs...

Many thanks!

**Comment by 5Aki1** [comment]
Hello, 

You need to provide a bedgraph and peak file for any type. HisTrader expects the peaks to have signal within them. Are you using peaks that were called from that bedgraph?

**Comment by CriticalSci** [comment]
Thanks!

I think I figured the issue. I was using a consensus peak BED file with the bedtools coverage signal directly calculated from the BAMs from each replicate which could have led to peaks without signal in some cases.

I am now using the peak file and the bedGraph fold change signal derived from MACS2 for each replicate and it seems to be working well.

Note that in my case I had to convert the signal from bigWig to bedGraph using the UCSC utilities as I am using the outputs from the ENCODE pipeline.

Regarding the signal file, can you confirm the fold change signal is the most appropriate rather than the pvalue signal?

**Comment by 5Aki1** [comment]
Yeah I think the fold change works better. Sorry for the delay. I hadn't seen your comment
